Makefile

```diff
@@ -46,7 +46,7 @@ header:
 dist: build
 	@rm -rf ./dist && mkdir -p ./dist
 	@echo "--> Creating an archive"
-	@tar czvf yorc.tgz yorc && echo "TODO: clean this part after CI update" &&  cp yorc yorc.tgz dist/
+	@tar czvf yorc-$(VERSION).tgz yorc && echo "TODO: clean this part after CI update" &&  cp yorc yorc-$(VERSION).tgz dist/
```

Comment:
Would be better if we quote `"yorc-$(VERSION).tgz"` to prevent word splitting. And I think we cloud remove the TODO now.